.github/workflows/pull_request.yml

```diff
@@ -25,6 +25,6 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
-        run: pip3 install -r requirements.txt && pip3 install codecov
+        run: pip3 install -r requirements.txt && pip3 install codecov && pip install .
```

Comment:
using pip anyway as `python setup.py install` leaves a `build/` directory that makes mypy unhappy.